src/software/SfM/main_ComputeFeatures.cpp

```diff
@@ -56,6 +60,9 @@ int main(int argc, char **argv)
   std::string sImage_Describer_Method = "SIFT";
   bool bForce = false;
   std::string sFeaturePreset = "";
+#ifdef OPENMVG_USE_OPENMP
+  int iNumThreads = 0;
```

Comment:
Having the default value equals to 1 would be preferable (default thread count should be a positive integer)?